packages/marshal/src/helpers/internal-types.js

```diff
@@ -5,13 +5,6 @@
 
 import '../types.js';
```

Comment:
This `import` is the reason your "Why do I need these?" explicit imports are necessary.  Any JS in a JSDoc types file requires you to explicitly use `import(./internal-types.js).Foo` in the other JS files.  Without any JS statements, the types becomes ambient.

When we move from ambient types to explicit typing, you will need those explicit imports, but until we bite that bullet, you can just remove this line and the explicit imports.

Reply:
Thanks! Done.